Test/handler_test.py

```diff
@@ -16,7 +17,17 @@ def setUp(self):
             pass
 
     def tearDown(self):
-        os.remove(self.fifo_pipe_path)
+        if os.name == "posix":
+            os.remove(self.fifo_pipe_path)
```

Comment:
In the event that we're on a non-linux system, are we just skipping anything related to the algorithm handler? It might be easier just to make the HandlerTest objects themselves optional in that case.

Reply:
Handler was running just fine on windows it was only the pipe that needed changing. Based on what I was seeing it seemed that all it does is pipe the results back to the unit test. If that is correct, can I just use stdin as that pipe for windows?